apps/extension/src/background/keyring/keyring.ts

```diff
@@ -408,6 +408,22 @@ export class KeyRing {
     return accounts.find((acc) => acc.id === activeAccount?.id);
   }
 
+  public async updateDefaultAccount(address: string): Promise<void> {
+    const accounts = await this.queryAllAccounts();
+    const account = accounts.find((acc) => acc.address === address);
+    if (!account) {
+      throw new Error(`Account with address ${address} not found.`);
+    }
+    if (
+      account.type === AccountType.PrivateKey ||
```

Comment:
I think we can remove this - I imported a Private key, so no available mnemonic, and I was able to use this as my "default" account, however, I was unable to later "switch" to it after setting something else.